setup.py

```diff
@@ -26,7 +26,8 @@
     long_description = fh.read()
 
 install_requires = [
-    "craft-application~=1.0",
+    # craft-application~=1.1",
+    "craft-application@git+https://github.com/canonical/craft-application.git@aml/1.1-candidate",
```

Comment:
We are going to use the git version?

Reply:
We need to release craft-application 1.1 and then change that over before merging to main.